doc/how_to/get_started.rst

```diff
@@ -189,64 +194,83 @@ a ``Probe`` and we don’t need to set it *manually*.
 
 
 
-.. image:: get_started_files/get_started_20_1.png
+.. image:: get_started_files/get_started_22_1.png
+
 
+If your recording does not have a ``Probe``, you can set it using
+``set_probe``. There is more information
```

Comment:
Maybe we should add a comment that this does not sets the probe of the current recorder but creates a copy?

Reply:
I think this is important. Did you finally get rid of `inplace`?

Reply:
Na, the issue is still there. 
https://github.com/SpikeInterface/spikeinterface/issues/2193

We never got Sam take on it.

Reply:
I've added a comment